src/extension.ts

```diff
@@ -142,6 +142,9 @@ function colorRuby(x: Parser.SyntaxNode, editor: vscode.TextEditor) {
 			colors.push([x, 'variable'])
 		} else if (x.type == 'call' && x.lastChild!.type == 'identifier') {
 			colors.push([x.lastChild!, 'entity.name.function'])
+		// Method parameters
+		} else if (x.type == 'identifier' && x.parent!.type == 'method_parameters') {
```

Comment:
This should maybe be moved to the end of control statement since `x.type == 'identifier'` will probably catch quite a few things.